proxy-node-shared/src/main/java/uk/gov/ida/notification/shared/logging/ProxyNodeLoggingFilter.java

```diff
@@ -29,25 +30,24 @@
 
     @Override
     public void filter(ContainerRequestContext requestContext) {
-        String istioTraceId = requestContext.getHeaderString(X_B3_TRACEID);
-        if (istioTraceId != null && !istioTraceId.isBlank()) {
-            MDC.put(X_B3_TRACEID, istioTraceId);
-        }
+        ofNullable(requestContext.getHeaderString(X_B3_TRACEID))
```

Comment:
Was there a reason to do this? This is much less clear so unless we have a good reason to use Optionals here this should be reverted. The `if` statement is more verbose but more clear and easier to understand.